src/utils/appInstaller/getAbiFromContentUri.ts

```diff
@@ -0,0 +1,30 @@
+import { ethers } from 'ethers'
+import fetch from 'node-fetch'
+import { AbiItem } from '~/src/types'
+import { getContentHash } from './utils'
+
+/**
+ * Gets the ABI from an Aragon App release directory
+ * @param contentURI
+ * @param options
+ */
+export default async function getAbiFromContentUri(
+  contentURI: string,
+  options: { ipfsGateway: string }
+): Promise<AbiItem[]> {
+  const { ipfsGateway } = options
+
+  const contentHash = ethers.utils.isHexString(contentURI)
+    ? getContentHash(contentURI)
+    : contentURI.split('/ipfs/')[1]
```

Comment:
We get the `contentHash` directly from the URI that we publish locally: https://github.com/aragon/buidler-aragon/pull/36/files#diff-b4e1f7bc0589bb21a4d487cba1d113c8R186

Reply:
About the string split `'/ipfs/'`, how should contentURI look like?

Reply:
Is what we define in the line I linked. Looks something like: `http:https://ipfs.eth.aragon.network/ipfs/<contentHash>`

Reply:
Improved with: https://github.com/aragon/buidler-aragon/pull/36/commits/c28cc2f695ea3f4a2bb6b0e090b7a7a40a0bf0c4